src/game.cpp

```diff
@@ -13308,7 +13308,7 @@ int game::slip_down_chance( climb_maneuver, climbing_aid_id aid_id,
 
     const climbing_aid &aid = aid_id.obj();
 
-    int slip = 100;
+    float slip = 100.0f;
```

Comment:
Far below this are the arithmetic operations that currently result in truncation.
```c++
slip /= 2;
slip *= wet_penalty;
slip /= std::max( 1, u.dex_cur + u.str_cur );
slip /= u.get_modifier( character_modifier_slip_prevent_mod );
slip /= std::max( stamina_ratio, .1f );
```